dev/serializer_context.h

```diff
@@ -19,6 +19,9 @@ namespace sqlite_orm {
             serializer_context(const db_objects_type& dbObjects) : db_objects{dbObjects} {}
         };
 
+        //template<class DBOs>
+        //serializer_context(const DBOs&) -> serializer_context<DBOs>;
```

Comment:
why did you leave a comment? we'd better skip commented out code cause it is useless during code reading and we can use version control system if we want to keep some unused code

Reply:
Thanks for catching this as well!